includes/validation/class-amp-validation-manager.php

```diff
@@ -262,7 +262,7 @@ public static function post_supports_validation( $post ) {
 		}
 
 		// Story post type always supports validation.
-		if ( AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG === $post->post_type ) {
+		if ( AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG === $post->post_type || AMP_Story_Post_Type::POST_TYPE_SLUG === $post->post_type ) {
```

Comment:
I had this in my original PR for post type.

Reply:
```suggestion
		if ( in_array( [ AMP_Story_Legacy_Post_Type::POST_TYPE_SLUG, AMP_Story_Post_Type::POST_TYPE_SLUG ], $post->post_type, true ) ) {
```

Reply:
Updated within 178f07b